src/services/auth.ts

```diff
@@ -1,8 +1,14 @@
 import { AxiosResponse } from 'axios';
 import { AccountResponse, AuthLocalResponse } from '../types';
-import axios, { setAuthHeader } from './axios';
+import axios from './axios';
 
-export const authDetailsKey = '__auth_details__';
+export interface AuthDetails {
+    session: AuthLocalResponse['data']['attributes'];
+    user?: AccountResponse['data']['attributes'];
+}
+
+export const sessionStorageKey = '__auth_session__';
+export const userStorageKey = '__auth_user__';
```

Comment:
Storing the two items separate as I have a feeling this gives us the most flexibility for future changes.